src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -65,7 +65,10 @@ public class ModuleInst extends AbstractModuleInst<Module, Site, ModuleInst>{
     private ArrayList<Net> nets;
```

Comment:
Not sure whether this should be in the javadoc:
```suggestion
    // Note that these are references to nets inside 'design' that this ModuleInst
    // inserted and is exclusively responsible for. As such, any static nets present in
    // this list must be handled with care.
    private ArrayList<Net> nets;
```